pytorch_lightning/trainer/connectors/logger_connector/result.py

```diff
@@ -162,9 +162,9 @@ def __init__(self, metadata: _Metadata, is_tensor: bool) -> None:
         self.meta = metadata
         self.has_reset = False
         if is_tensor:
-            self.add_state("value", torch.tensor(0, dtype=torch.float))
+            self.add_state("value", torch.tensor(0, dtype=torch.float), dist_reduce_fx=torch.sum)
```

Comment:
why do we need this?

Reply:
We had a bug with distributed training and we didn't reduce properly.